Dockerfile

```diff
@@ -41,7 +41,15 @@ RUN : \
  && create-db.sh aplus aplus django-migrate.sh \
  && :
 
+RUN pip_install debugpy git+https://github.com/mher/flower#egg=flower
 
 WORKDIR /srv/aplus
 EXPOSE 8000
+EXPOSE 5555
+
+RUN mkdir -p /var/celery/results \
+ && chown aplus:nogroup /var/celery/results
+RUN mkdir -p /var/celery/results \
+ && chown aplus:nogroup /var/celery/results
```

Comment:
This can't be right. You create the same directory twice.

Reply:
Hmm... no idea what has happened here, but fixed